register.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Register</title>
+</head>
+<body>
+    <form action="">
+        <div class="container">
+          <h1>Sign Up</h1>
+          <p>Please fill in this form to create an account.</p>
+      
+          <label for="email"><b>Email:</b></label>
+          <input type="text" placeholder="Enter Email" name="email" required>
```

Comment:
add id instead of the name because the for work with id, not name 
replace that in all inputs